### PR TITLE
Style

### PR DESCRIPTION
**Pull request by janzeng**
I have put in a instruction section, removed the deal and remove areas, and finished up the card interactions so that users will know what their possible moves are.

### CODE REVIEW

**Comment by AntonSynytsiaOSU** [comment]
Good job with the OSU styling! I like it and it is very awesome!

There are a few issue though:
- In the master branch, whenever a card is hovered upon, if it can be removed it is hovered in one color, and if it can be removed it is hovered in another color. This foretold the player what would happen to the card when it is clicked. In your pull request, however, movable and removable cards are highlighted in the same color.
- In the master branch, selected, movable cards are also highlighted, to indicate which card the user selected.  In your commit, however, selected, movable regain their original, default state. Are these changes intended?

**Comment by janzeng** [comment]
Ah I see what you mean. I’ll tweak it a bit more and push a new commit, because I think then I could better tie in the instructions and possible moves.

**Comment by janzeng** [comment]
Alright, I changed it so the movable card stays highlighted. I kept the same highlighting for movable and removable cards though, because if a user has the option to remove a card that takes precedence over moving anyways. Thanks for the feedback!

**Comment by janzeng** [comment]
Here is a [screenshot](https://github.com/cs361fall2017/team2/blob/master/images/MoveCard.png) of what it looks like when you click on a movable card and then hover over the empty cell you are moving to.

File: sprint2/src/main/java/assets/css/AcesUp.css [hunk 1]
```diff
@@ -170,4 +191,20 @@ td{
     left: 18px;
     bottom: 50px;
     max-width: 125px;
+}
+
+#resetButton{
```
Comment (AntonSynytsiaOSU) [inline sprint2/src/main/java/assets/css/AcesUp.css]: This is a duplicate

**Comment by AntonSynytsiaOSU** [comment]
The highlighting of selected card is okay now. Thanks for fixing it.

In the commit where you added a reset button, you accidentally there is a duplicate resetButton CSS style.

Also to disable user from accidentally selecting html content, you can add these commands to the body:
```css
body {
    -webkit-touch-callout: none;
    -webkit-user-select: none;
    -khtml-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
    user-select: none;
}
```

**Comment by janzeng** [comment]
Okay I'm done messing with it for tonight haha. Sorry for all the commits!

**Comment by lcsfrey** [comment]
The styling looks really good. I like that you changed the card highlighting to match your style as well. One thing to note: Since the beaver icon covers the instructions when the game loads, it might be better to have the beaver icon in the bottom right since most of the right side is empty space.

**Comment by AntonSynytsiaOSU** [comment]
And don't forget to add this:
```css
body {
    -webkit-touch-callout: none;
    -webkit-user-select: none;
    -khtml-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
    user-select: none;
}
```
Just having, user-select will only work on particular browsers, but not on others: https://stackoverflow.com/questions/826782/how-to-disable-text-selection-highlighting-using-css

**Comment by lcsfrey** [comment]
Everything has been fixed. The logo no longer covers any of the game and the highlighting still works as expected. Good work Gabriel!

**Pull request merged**